client/src/Pages/Dashboard/DashboardView.js

```diff
@@ -87,4 +87,4 @@ export default function DashboardView(props) {
       </>
     );
   }
-}
+}
```

Comment:
can you change your editor settings to insert a new line at the end of the file?